### PR TITLE
Testnet changes

### PR DESCRIPTION
**Pull request by ZainAliSyed24**
- Test net changes merging to master till **commit 4ea17be**

### CODE REVIEW

**Review by muursh — COMMENTED** [review]
I have no real qualms here. 1) is the app-release.aab binary needed? 2) I haven't run this so won't approve right now but no real issues from my side

**Comment by ZainAliSyed24** [comment]
> qualms

**app-release.aab** file is for play store uploading  only for making release to production or internal testing  .

**Comment by muursh** [comment]
Does it need to be on github though?

**Comment by ZainAliSyed24** [comment]
> eed to be on github th

I am going to  add this in gitignore file , so we don't have this in future , either  **apk** or .**abb** file both

File: components/TransactionsNavigationHeader.js [hunk 1]
```diff
@@ -166,20 +166,21 @@ export default class TransactionsNavigationHeader extends Component {
...
-    [
-      {
-        id: TransactionsNavigationHeader.actionKeys.Exchange,
-        text: loc.lnd.exchange,
-        icon: TransactionsNavigationHeader.actionIcons.Exchange,
-      },
-    ],
   ];
 
+  // {
```
Comment (renie) [inline components/TransactionsNavigationHeader.js:171]: Remove comments, please.

File: components/WalletsCarousel.js [hunk 2]
```diff
@@ -212,11 +209,11 @@ const WalletCarouselItem = ({ item, index, onPress, handleLongPress, isSelectedW
           onPressedOut();
         }}
       >
-        {/* <LinearGradient shadowColor={colors.shadowColor} colors={WalletGradient.gradientsFor(item.type)} style={iStyles.grad}>
-      <Image source={image} style={iStyles.image} /> */}
-        <View style={[iStyles.grad, { backgroundColor: colors.walletBalanceBgColor }]}>
-          <Text style={iStyles.br} />
+        <LinearGradient shadowColor={colors.shadowColor} colors={WalletGradient.gradientsFor(item.type)} style={iStyles.grad}>
```
Comment (renie) [inline components/WalletsCarousel.js:212]: Remove comments, please.

File: components/WalletsCarousel.js [hunk 3]
```diff
@@ -251,8 +248,9 @@ const WalletCarouselItem = ({ item, index, onPress, handleLongPress, isSelectedW
             {/* <Text style={[iStyles.btcInfo, { color: colors.inverseForegroundColor, fontFamily: type.semiBold }]}>{'571915' + '    '}</Text>
```
Comment (renie) [inline components/WalletsCarousel.js:248]: Remove comments, please.

File: ios/BlueWallet/Info.plist [hunk 4]
```diff
@@ -209,11 +209,11 @@
 		<string>armv7</string>
 	</array>
 	<key>UISupportedInterfaceOrientations</key>
-	<array>
+	<!-- <array>
```
Comment (renie) [inline ios/BlueWallet/Info.plist:212]: Remove comments, please.

File: ios/BlueWallet/Info.plist [hunk 5]
```diff
@@ -209,11 +209,11 @@
...
 		<string>UIInterfaceOrientationPortrait</string>
 		<string>UIInterfaceOrientationPortraitUpsideDown</string>
-	</array>
-	<key>UISupportedInterfaceOrientations~ipad</key>
+	</array> -->
```
Comment (renie) [inline ios/BlueWallet/Info.plist:215]: Remove comments, please.

File: screen/wallets/add.js [hunk 6]
```diff
@@ -245,14 +252,13 @@ const WalletsAdd = () => {
         </View>
         {/* <BlueFormLabel>{loc.wallets.add_wallet_type}</BlueFormLabel> */}
         <View style={styles.buttons}>
-          <BitcoinButton testID="ActivateBitcoinButton" active={selectedWalletType === ButtonSelected.ONCHAIN} onPress={handleOnBitcoinButtonPressed} style={styles.button} />
+          {isTestMode ? <BitcoinButton testID="ActivateBitcoinButton" active={selectedWalletType === ButtonSelected.ONCHAIN} onPress={handleOnBitcoinButtonPressed} style={styles.button} testnet /> : <BitcoinButton testID="ActivateBitcoinButton" active={selectedWalletType === ButtonSelected.ONCHAIN} onPress={handleOnBitcoinButtonPressed} style={styles.button} />}
           {/* <BitcoinButton testID="ActivateBitcoinButton" active={selectedWalletType === ButtonSelected.ONCHAIN} onPress={handleOnBitcoinButtonPressed} style={styles.button} /> */}
```
Comment (renie) [inline screen/wallets/add.js:256]: Remove comments, please.

File: screen/wallets/add.js [hunk 7]
```diff
@@ -245,14 +252,13 @@ const WalletsAdd = () => {
...
 
-          {/* <LightningButton active={selectedWalletType === ButtonSelected.OFFCHAIN} onPress={handleOnLightningButtonPressed} style={styles.button} />
+          <LightningButton active={selectedWalletType === ButtonSelected.OFFCHAIN} onPress={handleOnLightningButtonPressed} style={styles.button} />
           {backdoorPressed > 10 ? <LdkButton active={selectedWalletType === ButtonSelected.LDK} onPress={handleOnLdkButtonPressed} style={styles.button} subtext={LightningLdkWallet.getPackageVersion()} text="LDK" /> : null}
-          <VaultButton active={selectedWalletType === ButtonSelected.VAULT} onPress={handleOnVaultButtonPressed} style={styles.button} /> */}
-        </View>
-        <View style={styles.buttons}>
+          {/* <VaultButton active={selectedWalletType === ButtonSelected.VAULT} onPress={handleOnVaultButtonPressed} style={styles.button} /> */}
```
Comment (renie) [inline screen/wallets/add.js:260]: Remove comments, please.

File: screen/wallets/list.js [hunk 8]
```diff
@@ -235,8 +235,8 @@ const WalletsList = () => {
   const renderSectionFooter = (section) => {
     switch (section.section.key) {
       case WalletsListSections.TRANSACTIONS:
-        // if (dataSource.length === 0 && !isLoading) {
-        if (dataSource.length === 1 && !isLoading) {
+        if (dataSource.length === 0 && !isLoading) {
+          // if (dataSource.length === 1 && !isLoading) {
```
Comment (renie) [inline screen/wallets/list.js:239]: Remove comments, please.

File: screen/wallets/transactions.js [hunk 9]
```diff
@@ -241,8 +241,8 @@ const WalletTransactions = () => {
 
            */}
           {wallet.getTransactions().length > 0 && wallet.chain !== Chain.OFFCHAIN && wallet.type !== LightningLdkWallet.type && renderSellFiat()}
-          {wallet.chain === Chain.OFFCHAIN && renderMarketplaceButton()}
-          {wallet.chain === Chain.OFFCHAIN && Platform.OS === 'ios' && renderLappBrowserButton()}
+          {/* wallet.chain === Chain.OFFCHAIN && renderMarketplaceButton() */}
```
Comment (renie) [inline screen/wallets/transactions.js:244]: Remove comments, please.

File: screen/wallets/transactions.js [hunk 10]
```diff
@@ -323,11 +323,11 @@ const WalletTransactions = () => {
   };
 
   const renderSellFiat = () => {
-    return (
-      <TouchableOpacity accessibilityRole="button" onPress={navigateToBuyBitcoin} style={[styles.marketplaceButton2, stylesHook.marketplaceButton2]}>
-        <Text style={[styles.marketpalceText1, stylesHook.marketpalceText1]}>{loc.wallets.list_tap_here_to_buy}</Text>
-      </TouchableOpacity>
-    );
+    // return (
```
Comment (renie) [inline screen/wallets/transactions.js:326]: Remove comments, please.

**Review by renie — CHANGES_REQUESTED** [review]
Just very minor stuff.
Good job!

**Review by renie — APPROVED** [review]

**Pull request merged**